lms/templates/courseware/info.html

```diff
@@ -14,50 +14,55 @@
-    <div class="msg msg-reverify is-dismissable">
-      <div class="msg-content">
-        <h2 class="title">${_("You are not enrolled yet")}</h2>
-        <div class="copy">
-          <p class='enroll-message'>
-            ${_(u"You are not currently enrolled in this course. {link_start}Sign up now!{link_end}").format(
-            link_start=u"<a href={}>".format(url_to_enroll),
-            link_end=u"</a>"
-            )}
-          </p>
-        </div>
+<div class="wrapper-msg urgency-low" id="failed-verification-banner">
+  <div class="msg msg-reverify is-dismissable">
+    <div class="msg-content">
+      <h2 class="title">${_("You are not enrolled yet")}</h2>
+      <div class="copy">
+        <p class='enroll-message'>
+          ${_(u"You are not currently enrolled in this course. {link_start}Sign up now!{link_end}").format(
+          link_start=u"<a href={}>".format(url_to_enroll),
+          link_end=u"</a>"
+          )}
+        </p>
       </div>
     </div>
   </div>
+</div>
 % endif
 
 <%include file="/courseware/course_navigation.html" args="active_page='info'" />
 
 <%block name="js_extra">
-  <script type="text/javascript" src="${static.url('js/jquery.treeview.js')}"></script>
-  <script type="text/javascript" charset="utf-8">
-$(document).ready(function(){
-    $(".handouts").treeview({collapsed:true, unique:true/*, cookieId: "treeview-book-nav", persist: "cookie"*/});
-    });
-  </script>
+<script type="text/javascript" src="${static.url('js/jquery.treeview.js')}"></script>
+<script type="text/javascript" charset="utf-8">
+ $(document).ready(function(){
+     $(".handouts").treeview({collapsed:true, unique:true/*, cookieId: "treeview-book-nav", persist: "cookie"*/});
+ });
+</script>
 </%block>
 
 <%block name="bodyclass">view-in-course view-course-info ${course.css_class or ''}</%block>
 <section class="container">
+  <div class="welcome-wrapper">
+    <section class="welcome">
+        <h1>${_("Welcome to {org}'s {course_name}!").format(org=course.id.org, course_name=course.id.course) | h}</h1>
+        <h2>${course.display_name | h}</h2>
+    </section>
+  </div>
   <div class="info-wrapper">
     % if user.is_authenticated():
     <section class="updates">
       % if studio_url is not None and masquerade and masquerade.role == 'staff':
-        <div class="wrap-instructor-info studio-view">
-          <a class="instructor-info-action" href="${studio_url}">
-            ${_("View Updates in Studio")}
-          </a>
-        </div>
+      <div class="wrap-instructor-info studio-view">
+        <a class="instructor-info-action" href="${studio_url}">
+          ${_("View Updates in Studio")}
+        </a>
+      </div>
       % endif
 
-      <h1>${_("Course Updates &amp; News")}</h1>
+      <h1>${_("Course Updates and News")}</h1>
       ${get_course_info_section(request, course, 'updates')}
     </section>
     <section aria-label="${_('Handout Navigation')}" class="handouts">
```

Comment:
@marcotuts The plan is to move the Handout Navigation section as part of https://openedx.atlassian.net/browse/ECOM-2808. I'll leave this as-is for now, and make the changes as part of that PR.